activemq_xml/datadog_checks/activemq_xml/data/conf.yaml.example

```
@@ -45,3 +60,19 @@ instances:
   #   - <SUBSCRIBER_1>
   #   - <SUBSCRIBER_2>
 
+  ## @param skip_proxy - boolean - optional - default: false
+  ## The (optional) skip_proxy parameter bypasses any proxy
```

```suggestion
  ## Set `skip_proxy` parameter to true to bypasses any proxy
```

Just to be sure, this means that this parameter can be set in the `init_config` to be applied to all instances, or just at an instance level? If there is both, who wins?

---

instance